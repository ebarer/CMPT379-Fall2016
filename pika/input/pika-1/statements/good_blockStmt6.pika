exec {
	var t := 10.
	var s := "Mister".
	{
		var s := 12.
		print "They";.
		{
			t := s.
			const s := "me".
			{
				print "call";.
			}
			print s;.
		}
	}
	print s;.
	print "Tibbs!", _n_.
	print t, _n_.
}
