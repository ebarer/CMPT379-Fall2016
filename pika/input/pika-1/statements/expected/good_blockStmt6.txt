They call me Mister Tibbs!
12
